gymnasium/core.py

```diff
@@ -272,15 +275,35 @@ def __init__(self, env: Env[ObsType, ActType]):
         self._cached_spec: EnvSpec | None = None
 
     def __getattr__(self, name: str) -> Any:
-        """Returns an attribute with ``name``, unless ``name`` starts with an underscore."""
+        """Returns an attribute with ``name``, unless ``name`` starts with an underscore.
+
+        Args:
+            name: The variable name
+
+        Returns:
+            The value of the variable in the wrapper stack
+
+        Warnings:
+            This feature is deprecated and removed in v1.0 and replaced with `env.get_attr(name})`
+        """
         if name == "_np_random":
             raise AttributeError(
                 "Can't access `_np_random` of a wrapper, use `self.unwrapped._np_random` or `self.np_random`."
             )
         elif name.startswith("_"):
             raise AttributeError(f"accessing private attribute '{name}' is prohibited")
+        logger.warn(f"This feature is deprecated and will be removed in v1.0 and replaced with `env.get_attr('{name}')`.")
```

Comment:
We should probably promote just doing `env.name` or `env.unwrapped.name`, since that covers like 99.9% of use-cases